spec/lib/rspec/scaffold_spec.rb

```diff
@@ -18,24 +18,73 @@
   end
 
   describe ".testify_file(filepath, mode=:to_file)" do
-    let(:path) { klass.root + "spec/fixtures/classes/ability.rb" }
+    let(:path) { FIXTURE_ROOT.join('models/activity_feature.rb').to_s }
+    let(:same_dir_spec_path) { path.to_s.gsub(%r'\.rb\z', '_spec.rb') }
+    let(:custom_output_path) { RSpec::Scaffold.root.join("spec/fixtures/spec/models/activity_feature_spec.rb").to_s }
 
-    context "when passed a single file and :to_text mode" do
-      xit "should puts about action taken and return the scaffold as text" do
-        expect(0).to eq 1
+    context "when passed :to_text mode" do
+      it "should return the scaffold as a String object" do
+        expect(described_class.testify_file(path)).to eq models_activity_feature_rb_test_scaffold
+      end
+    end
+
+    context "when passed :to_file mode with no output path" do
+      it "should puts about where test scaffold was saved to, output to a file (in the same directory as original) and return the path" do
+        FakeFS.with_fresh do
```

Comment:
FakeFS helps with testing changes in filesystem.
It is not without its problems, but seems to work fine here.